_static/css/aiven.css

```diff
@@ -44,169 +20,29 @@
   src: url(../fonts/Inter-Medium.ttf);
 }
 
-body {
-  background-color: var(--bg);
-}
 
 .sidebar-drawer {
   border-right: none;
-  background: var(--sideBar);
-}
-
-.toc-drawer {
-  background-color: var(--bg);
 }
 
 h1,
 h2,
 h3 {
-  color: var(--primaryColor);
+  color: var(--color-foreground-primary);
   font-family: InterBold;
 }
 
 p {
-  color: var(--secondaryColor);
+  color: var(--color-content-foreground);
   font-family: Inter;
 }
 
 a {
-  color: var(--link) !important;
+  color: var(--color-link);
```

Comment:
For dark mode this line sets the color to be red by default, which is different than befor. 

I see it references --color-link: #ff3554; While in the old version in was  --linkDark: #d2d2d6;
The hover colors also changed to --color-brand-content: #368ce2; while it was --linkDark: #d2d2d6; before (that's where the !important was enforcing it to be same color as non-hover)

Reply:
<img width="1136" alt="Screenshot 2022-07-03 at 21 20 19" src="https://user-images.githubusercontent.com/4600541/177054279-a8ef1c84-6355-4f80-8799-368d1c62a3ae.png">

Reply:
This helps, think I fixed it now.